content/shmem_query_thread.tex

```diff
@@ -0,0 +1,33 @@
+\apisummary{
+Returns the level of thread support provided by the library.}
+
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void shmem_query_thread(int *provided);
+\end{Csynopsis}
+
+\begin{apiarguments}
+\apiargument{OUT}{provided}{The thread level support provided by the \openshmem{} implementation.}
+\end{apiarguments}
+
+
+\apidescription{
+The \FUNC{SHMEM\_QUERY\_THREAD} call returns the level of thread support
+currently being provided. The value returned will be same as \VAR{provided}
+returned in the \FUNC{SHMEM\_INIT\_THREAD}, if the \openshmem{} library was
+initialized by \FUNC{SHMEM\_INIT\_THREAD}.
```

Comment:
I think it should be any one of the four modes, not just `SINGLE.` If an implementation wants to provide `SERIALIZED`, `FUNNELED`, or `MULTIPLE` as a default, why constraint it.